cmd/kured/main.go

```diff
@@ -93,6 +96,10 @@ func init() {
 }
 
 func main() {
+	os.Stdout.WriteString(fmt.Sprintf("Command line args: %v\n", os.Args))
+	os.Stderr.WriteString(fmt.Sprintf("Command line args: %v\n", os.Args))
```

Comment:
Is that really worth sending into stderr?

Reply:
removed, that was for debugging :)